py/server/deephaven/parquet.py

```diff
@@ -19,7 +19,7 @@
 _JFile = jpy.get_type("java.io.File")
 _JCompressionCodecName = jpy.get_type("org.apache.parquet.hadoop.metadata.CompressionCodecName")
 _JParquetInstructions = jpy.get_type("io.deephaven.parquet.table.ParquetInstructions")
-_JS3Instructions = jpy.get_type("io.deephaven.parquet.table.S3Instructions")
+_JS3Instructions = jpy.get_type("io.deephaven.extensions.s3.S3Instructions")
```

Comment:
We may want to move python logic to its own python module b/c S3Instructions may not exist on classpath. I've done as much with sql.py for the same reason.

I'd very much like to have tighter coupling between python modules that adapt java modules, but that's not something we've worked towards before (for example, packaging the s3 python code inside the s3 jar).